src/Cli.Tests/FileSystemUtils.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.IO.Abstractions.TestingHelpers;
+using System.Reflection;
+
+namespace Cli.Tests;
+
+internal static class FileSystemUtils
+{
+    public static MockFileSystem ProvisionMockFileSystem()
+    {
+        MockFileSystem fileSystem = new();
+
+        fileSystem.AddFile(
+            fileSystem.Path.Combine(
+                fileSystem.Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location) ?? "",
+                "dab.draft.schema.json"),
+            new MockFileData("{ \"additionalProperties\": {\"version\": \"https://github.com/Azure/data-api-builder/releases/download/vmajor.minor.patch/dab.draft.schema.json\"} }"));
```

Comment:
Why is this needed? I see it was there before, would be nice to have a comment why we add the additional property here

Reply:
I knew there was a reason but I didn't remember what that reason was, so I deleted it and ran the tests!

The reason is that this method gets called: https://github.com/Azure/data-api-builder/blob/main/src/Config/RuntimeConfigLoader.cs#L255

If we haven't pushed the file into the mock file system, it'll get a `FileNotFoundException` which traces back to https://github.com/Azure/data-api-builder/blob/main/src/Cli/ConfigGenerator.cs#L146

It use to not be a problem as we were doing disk IO but now that we aren't (it's a mock disk) we need to ensure that files we expect to be on disk _are_ on disk.

I'll add a comment to it.